spec/fixtures/custom_nginx.template

```diff
@@ -339,6 +341,19 @@ http {
             }
         }
 
+        location /xml {
+            content_by_lua_block {
+                local mu = require "spec.fixtures.mock_upstream"
+                local xml = [[
+                  <?xml version="1.0" encoding="UTF-8"?>
+                    <note>
+                      <body>Kong, the Monolith destroyer.</body>
```

Comment:
PS: the tagline is "Kong, monolith destroyer" (dropping the `the`), see https://github.com/Kong/kong/blob/b3583f98c5c1db03e3ddd683d26ac0daa5b63687/kong/cmd/roar.lua#L4 :wink: Can update at merge time.